### PR TITLE
Add/only show clock icon when draft scheduled

### PR DESCRIPTION
**Pull request by alvarezmelissa87**
### Purpose
Only if a post has a scheduled time → clock icon next to the post details
### Things to Note
Need to update snapshot.
### Review

**Some aspects to consider during review:**
* Functionality and implementation
* Architecture and performance
* Code readability and style

_A friendly reminder to add a reviewer/reviewers, add an assignee (yourself if you've worked on the change), set the code review status, and set the component request. :smile:_

### CODE REVIEW

File: ColabTool/Shared/Post/story.jsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ const draftDetails = {
   createdAt: 'on March 2nd at 12:45pm (GMT)',
   email: 'ash@buffer.com',
   isRetweet: false,
-  postAction: 'This post is scheduled for 9:42pm (GMT)',
+  postAction: 'This post will be added to the queue',
```
Comment (hharnisc) [inline ColabTool/Shared/Post/story.jsx]: Might be nice to create some demo post data and import it in these story files. That way we only need to change this in one place. (A task for a later followup commit)

File: ColabTool/Shared/PostFooter/index.jsx [hunk 2]
```diff
@@ -78,7 +78,15 @@ const renderEdit = ({
   );
 };
 
-const renderIcon = isPastDue => (isPastDue ? <WarningIcon color={'torchRed'} /> : <ClockIcon />);
+const renderIcon = ({ isPastDue, scheduledAt }) => {
```
Comment (hharnisc) [inline ColabTool/Shared/PostFooter/index.jsx]: There's an ambiguous state that makes it a little trickier to understand here. Returning null is a little clearer here I think :smile:
 
```js
const renderIcon = ({ isPastDue, scheduledAt }) => {
   if (isPastDue) {
     return (<WarningIcon color={'torchRed'} />);
   } else if (scheduledAt) {
     return (<ClockIcon />);
   }
   return null;
 };
```

File: ColabTool/Shared/Post/story.jsx (hunk 1 above)
Comment (alvarezmelissa87) [inline ColabTool/Shared/Post/story.jsx]: 😂  Yes! I definitely thought that while going through and changing all these in the same way! Sounds perfect for a follow up 😄

**Comment by alvarezmelissa87** [comment]
Heya @hharnisc - I've gone ahead and made the updates! Thanks so much for looking over this! 😄

**Comment by alvarezmelissa87** [comment]
Heya Team! Going to squash and merge this! 😄

**Pull request merged**